apps/createsamples/utility.cpp

```diff
@@ -1270,7 +1270,7 @@ void cvCreateTrainingSamples( const char* filename,
                 if( showsamples )
                 {
                     cvShowImage( "Sample", &sample );
-                    if( cvWaitKey( 0 ) == 27 )
+                    if( (cvWaitKey( 0 ) & 0xFF) == 27 )
```

Comment:
I think we could still use 27 for ESC value. Enum definition for ESC and a few other useful values only would be inconsistent while all other(and IMO most of) values included to the enum will be never used.